dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -1064,10 +1065,10 @@ public Entry getPublishedEntryByPath(@ApiParam(value = "repository path", requir
         security = @SecurityRequirement(name = JWT_SECURITY_DEFINITION_NAME))
     @ApiOperation(nickname = "getAllWorkflowByPath", value = "Get a list of workflows by path.", authorizations = {
         @Authorization(value = JWT_SECURITY_DEFINITION_NAME)}, notes = "Do not include workflow name.", response = Workflow.class, responseContainer = "List")
-    public List<Workflow> getAllWorkflowByPath(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user") @Auth User user,
-        @ApiParam(value = "repository path", required = true) @PathParam("repository") String path) {
+    public List<Workflow> getAllWorkflowByPath(@Parameter(hidden = true, name = "user") @Auth User user,
+        @Parameter(description = "repository path") @PathParam("repository") String path) {
         List<Workflow> workflows = workflowDAO.findAllByPath(path, false);
-        workflows.forEach(this::checkNotNullEntry);
+        checkNotNull(workflows, "Invalid repository path");
```

Comment:
`findAllByPath` returns `null` if the path is invalid (not the expected format).